Assets/Scripts/Projects/AssetUriFactory.cs

```diff
@@ -15,5 +15,10 @@ public static Uri CreateRemoteAssetUri(string url)
             
             return new Uri(url);
         }
+
+        public static Uri CreateRemoteAssetUri(Uri url)
```

Comment:
method naam iets wat misleidend?

Reply:
Deze naam is omdat deze methode hetzelfde gedrag moet vertonen als `CreateRemoteAssetUri(string url)` - namelijk een Uri teruggeven voor de opgegeven RemoteAssetUri. Ik denk wel dat een goede vraag is waarom deze functie bestaat en niet de IsRemote functie aangeroepen is op de Uri class 🤔

Reply:
Opgelost in volgende commit